openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -107,6 +107,9 @@ git apply "$root/openshift-knative-operator/hack/001-eventing-namespace-deletion
 # TODO: propose to upstream
 git apply "$root/openshift-knative-operator/hack/002-openshift-eventing-role.patch"
 
+# For SRVKE-629 we disable HPA:
```

Comment:
we should update the comment to refer to SRVKE-914 , as that is the new reason...

Reply:
ah, yeah. updated the file